demo/components/auto-complete.demo.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import {AutoComplete, AutoCompleteChangeEvent} from '../../src';
+
+export interface AutoCompleteDemoState {
+    open: boolean;
+    inputText: string;
+}
+
+const items = ['Muffins', 'Pancakes', 'Cupcakes', 'Souffles',
+               'Pasta', 'Soup', 'Caramel', 'Avazim', 'Moses'];
+
+export class AutoCompleteDemo extends React.Component<{}, AutoCompleteDemoState> {
+    public state = {open: true, inputText: ''};
```

Comment:
Why does the control in the demo start with an open popup? It looks strange.

Reply:
It's a leftover from when the caret wasn't there. Fixed it